Library/Homebrew/cmd/migrate.rb

```diff
@@ -0,0 +1,22 @@
+require "migrator"
+require "formula_renames"
+
+module Homebrew
+  def migrate
+    raise FormulaUnspecifiedError if ARGV.named.empty?
+
+    # Don't do anything if name is a symlink
+    ARGV.named.each do |name|
+      name = $3 if name =~ HOMEBREW_TAP_FORMULA_REGEX
+      unless (rack = HOMEBREW_CELLAR/name).exist? && !rack.subdirs.empty?
+        raise NoSuchKegError, name
+      end
+      raise "#{rack} is a symlink" if rack.symlink?
+    end
```

Comment:
@xu-cheng In `resolved_formulae` we have formulae but not the names passed to the command, can we get the name passed to the command using the formula?